### PR TITLE
Plotting graph with R v.s. Python

### PR DESCRIPTION
**Pull request by simonameng**
Community contribution from Qi Meng. It's about the codes for plotting some specific types of graphs we learnt in the lecture using R and Python.

### CODE REVIEW

File: r_python_graph_code.Rmd [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+
+In this document, I'll try to summerize some basic plots that we've learned in the course of `STAT GR5702 Exploratory Data Analysis and Visualization` and try to match the R codes with the corresponding Python codes. The examples provided in the document would be relatively simple. And I will introduce/use only some of the commonly used parameters in the examples. Hope the document can help people get some insights about which python packages/function to use if they want to create the plots. 
+
+The plots that I introduce in this document include `Histogram`, `Boxplot`, `Density Curve`, `Ridgeline Plot`, `QQ Plot`, `Scatter Plot`, `Heatmap`, `Parallel Coordinate Plots` and `Mosaic Plot`.
+
+In this document, I referred to some codes used in the lecture and the problem sets. There are also some python codes that I referred online have been cited at the bottom of each section.
+
+## Histogram
+
+------
```
Comment (Abhimanyu210100) [inline r_python_graph_code.Rmd:30]: Remove all the '------' lines from the .Rmd file

Comment (simonameng) [reply r_python_graph_code.Rmd]: '------' lines removed.

**Comment by alexnsaunders** [comment]
@Abhimanyu210100 I haven't checked the underlying file but looks like this is ready to check and merge if all ok.

**Comment by Abhimanyu210100** [comment]
@alexnsaunders I have merged the file

**Comment by Abhimanyu210100** [comment]
@simonameng Thank you for your contribution 🎉

**Pull request merged**